html-forms.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    </head>
```

Comment:
c'mon, at least title in the <head> should be

Reply:
added title ))
![image](https://github.com/shurachen23/Tekwill-Course/assets/121938064/7b9eacd5-a9e2-49a5-878d-7c2bd865fdb1)